summit/auxtel-vent-gates/launch_controller

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Launch the vent gate controller
+
+cd docker-compose-ops/summit
+source setup.env
+cd auxtel-vent-gates
+docker compose pull lsstts/vents-controller-aarch64:${CYCLE}
```

Comment:
Don't use the image here. Use the container name. `${CYCLE}` will not be present at the script level anyway.